MusicLister/MusicLister/Views/SongTableViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SongTableViewCell.swift
+//  MusicLister
+//
+//  Created by Alberto Camacho on 7/22/19.
+//  Copyright © 2019 Alberto Camacho. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SongTableViewCell: UITableViewCell {
+    static let identifier = String(describing: SongTableViewCell.self)
+    @IBOutlet weak var trackName: UILabel!
```

Comment:
Consider naming your variables "name", "artist", "album" etc. You're in the context to understand that the vars are describing a song. Also, you wrote 'Song' in the class name, but the variables say 'track'